backend/dataall/cdkproxy/assets/lakeformationdefaultsettings/index.py

```diff
@@ -66,4 +85,43 @@ def on_update(event):
 
 
 def on_delete(event):
-    pass
+    """"Removes the PivotRole from the existing Data Lake Administrators
+    Before adding any principal, it validates it exists if it is an IAM role
+    """
+    AWS_ACCOUNT = os.environ.get('AWS_ACCOUNT')
```

Comment:
I think we can also move the code for reading aws account and region env variables in each method to do it outside of the  handler to do it just once and not repeat ourself (DRY)